testRequests.py

```diff
@@ -107,8 +107,13 @@ def getTestScript(PrepID, nEvents):
         inputfile = open(filename, 'r')
         outputfile = open(tmpfilename, 'w')
         for line in inputfile:
-            outputfile.write(re.sub('(.*RAWSIM.*-n) \d*( .*)', r'\1 %s\2'
-                                    % (nEvents), line))
+            outline = re.sub('(.*--eventcontent LHE.*-n) \d*( .*)', r'\1 %s\2'
+                                    % (nEvents), line)
+            outline = re.sub('(.*--eventcontent DQM.*-n) \d*( .*)', r'\1 %s\2'
```

Comment:
Could you remind me where the cmsDriver.py commands this modifies appears. Were they in old wmLHE requests? I don't see them now. I think this is fine to include, I just want to know what command is being modified.